src/server/pegasus_server_impl.cpp

```diff
@@ -2618,5 +2618,57 @@ void pegasus_server_impl::release_db()
     _db = nullptr;
 }
 
+std::string pegasus_server_impl::dump_write_request(dsn::message_ex *request)
```

Comment:
```suggestion
static std::string dump_write_request(dsn::message_ex *request)
```

I see no private member of pegasus_server_impl is used, so make it a static function.